test/support/rtcp_fixtures.ex

```diff
@@ -1,22 +1,11 @@
 defmodule Membrane.RTCP.Fixtures do
   @moduledoc false
 
-  @external_resource "test/fixtures/rtcp/single_packet.hex"
-  @sample_rtcp_packet File.read!(@external_resource)
-
-  @external_resource "test/fixtures/rtcp/packets.hex"
```

Comment:
It shouldn't be deleted as it instructs the compiler to depend on that file, i.e. module `Membrane.RTCP.Fixtures` should be recompiled in case any of these files change.
See https://hexdocs.pm/elixir/Module.html#module-external_resource

Reply:
Good to know, I restored the old version